x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/dimension_editor.scss

```diff
@@ -15,6 +15,10 @@
   column-gap: $euiSizeXL;
 }
 
+.lnsIndexPatternDimensionEditor__operation {
+  position: static;
```

Comment:
What makes the item disappear is the parent `column-count: 2` passed to the `ul`. The column-count only works properly with block containers: https://drafts.csswg.org/css-multicol-1/#cc.

The list items are not respecting this rule and are using a `display: flex`. 

So to fix this bug you can override the `display` by adding:

```suggestion
  // overrides the .euiListGroupItem display flex. The parent container has a column-count property that only works properly with block elements. 
  display: block;
```


As you can see the bug now seems to be gone. The `position: static;` seemed to fix the issue but it was not positioning the item in the expected position.

<img width="626" alt="display_ block@2x" src="https://user-images.githubusercontent.com/2750668/104953284-035cbe00-59be-11eb-9188-d405c53e09e1.png">

**Note:** when we're fixing CSS bugs it's important to leave a comment explaining why we're adding a line of CSS that apparently doesn't seem to make sense to whoever looks at the code.